### ISSUE TITLE
get_cumulative_edgelist message unnecessary timestep 1

### ISSUE THREAD

**Issue by smjenness** [EpiModel/EpiModel#838]
@AdrienLeGuillou : I believe this is a new addition related to the requested `get_partners` warning. I find that the message generated when creating the cumulative edgelist in the init module at time step 1 unnecessary. Specifically, this is what we get when running any new model:

```r
At timestep = 1:
    the cumulative edgelist for network '1' is created from scratch 
    ** if this message repeats, check that the  
'    ** `update_cumulative_edgelist` function 
    ** is called after the networks resimulation. 



At timestep = 1:
    the cumulative edgelist for network '2' is created from scratch 
    ** if this message repeats, check that the  
'    ** `update_cumulative_edgelist` function 
    ** is called after the networks resimulation. 



At timestep = 1:
    the cumulative edgelist for network '3' is created from scratch 
    ** if this message repeats, check that the  
'    ** `update_cumulative_edgelist` function 
    ** is called after the networks resimulation. 
```

While not a bug, I find these this message is flagging a potential problem that doesn't exist in my case (i.e., I don't see the message repeating because the update function is called correctly). The warning should only appear in the case that the cumulative edgelist does not exist or not storing data correctly, so can we skip this initialization message?

**Comment by AdrienLeGuillou** [comment]
It is possible to remove them.
But that implies a lot of boilerplate code to check that timestep is not 1 or not `start` in case of a restarting simulation.
I will try to find a simple way to do it.

**Comment by AdrienLeGuillou** [comment]
No you are completely right sorry.
I don't know why that would be complex. And with `&& get_current_timestep(dat) > 1`, the second test would only occurs if the first returns `TRUE`. So that's not even using resources.

Should I make it `if (length(dat[["el.cuml"]]) < network & get_current_timestep(dat) > get_control(dat, "start"))` ? So it does not fire on restart if the `el_cuml` is triggered there ?

**Comment by smjenness** [comment]
That sounds right. It doesn't need to be checked again for restarted simulations, assuming that it would have been caught in the initial simulation starting from t1.

**Comment by AdrienLeGuillou** [comment]
fixed by #842
